crypto/x509/x509_lu.c

```diff
@@ -850,8 +850,7 @@ STACK_OF(X509) *X509_STORE_CTX_get1_certs(X509_STORE_CTX *ctx,
         if (objs != NULL)
             idx = x509_object_idx_cnt(objs, X509_LU_X509, nm, &cnt);
     } else {
-        objs = store->objs;
-        idx = x509_object_idx_cnt(objs, X509_LU_X509, nm, &cnt);
+        idx = x509_object_idx_cnt(store->objs, X509_LU_X509, nm, &cnt);
     }
     if (idx < 0 || objs == NULL) {
```

Comment:
This does not look correct. `objs` is checked here and you removed the assignment.

Reply:
My understadning is as follows. objs is NULL (set at 840) here we count number of indexes as a reader on the store..

it line 855 we take branch because objs is NULL (we take the branch always for ->objs case perhaps it intentional I just don't know (yet)